CodeMaid/UI/Dialogs/Options/Cleaning/CleaningUpdateViewModel.cs

```diff
@@ -52,6 +52,367 @@ public CleaningUpdateViewModel(CodeMaidPackage package, Settings activeSettings)
 
         #endregion Overrides of OptionsPageViewModel
 
+        #region Binding properties for variable header management
```

Comment:
Generally, this segment is massive (744 lines) and as far as I can tell it's all just to implement the ability to inject a header into the text box at the cursor location so the user doesn't have to type it or paste it from the clipboard?  It is a lot of code and all of the hard-coded indices are subject to break when/if we add more languages.  To be honest I think it would be better to just include the list of available variables as an informational message below the tabs and remove this code.  People have to type all of the content for the file header anyways, so it's actually probably slower to switch back and forth from the keyboard, to the mouse, and back to the keyboard.